doc/mdc-migration.md

```diff
@@ -0,0 +1,54 @@
+# MDC Android migration
```

Comment:
Can we also not use the `MDC` acronym unless we have expanded on it in one place atleast?

Reply:
I expanded it in the context part, but you're right I will use the full name in the title it self.